examples/amp-web-push.amp.html

```diff
@@ -0,0 +1,71 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>amp-web-push example</title>
+  <link rel="canonical" href="amps.html">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <style amp-custom>
+    .amp-web-push button {
+      border-radius: 2px;
+      border: 0;
+      box-sizing: border-box;
+      margin: 0;
+      padding: 10px 15px;
+      cursor: pointer;
+      outline: none;
+      font-size: inherit;
+      font-weight: inherit;
+      background: #2196F3;
+      color: white;
+      display: inline-block;
+      box-shadow: rgba(0, 0, 0, 0.12) 0px 1px 6px, rgba(0, 0, 0, 0.12) 0px 1px 4px;
+      text-align: center;
+      -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+    }
+
+    .amp-web-push button:hover {
+      background: #269BF8;
+    }
+
+    .amp-web-push button:active {
+      background: #1C91EE;
+      transform: scale(0.99);
+    }
+  </style>
+  <script async custom-element="amp-web-push" src="https://cdn.ampproject.org/v0/amp-web-push-0.1.js"></script>
+
+  <!-- AMP Web Push Extension Configuration
+  -->
+  <script id="amp-web-push" type="application/json">
```

Comment:
I replaced the `amp-access`-like JSON-based config with your suggested element-based config.

I initially went with the JSON-based config because there was an example from `amp-access`.